.github/actions/javascript/createOrUpdateStagingDeploy/createOrUpdateStagingDeploy.ts

```diff
@@ -1,14 +1,16 @@
-const fs = require('fs');
-const format = require('date-fns/format');
-const _ = require('underscore');
-const core = require('@actions/core');
-const CONST = require('../../../libs/CONST');
-const GithubUtils = require('../../../libs/GithubUtils');
-const GitUtils = require('../../../libs/GitUtils').default;
+import core from '@actions/core';
+import format from 'date-fns/format';
+import fs from 'fs';
+import CONST from '@github/libs/CONST';
+import GithubUtils from '@github/libs/GithubUtils';
+import type {StagingDeployCashData} from '@github/libs/GithubUtils';
+import GitUtils from '@github/libs/GitUtils';
+import type {EmptyObject} from '@src/types/utils/EmptyObject';
 
-async function run() {
+async function run(): Promise<Awaited<ReturnType<typeof GithubUtils.octokit.issues.create>>['data'] | void> {
```

Comment:
Let's extract `Awaited<ReturnType<typeof GithubUtils.octokit.issues.create>>['data']` to a separate type?